.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "markdown_metayaml"]
```

Comment:
I did the work on this a while ago so I might be misremembering, but I think I tried to do that exact thing and ended up needing to write [this](https://github.com/joachimneu/pelican-md-yaml/pull/5/files#diff-192340b2665e070a03ab506983bd2b13R16) to allow Pelican to get the correct data from the Markdown extension.

Just adding an extension to the Markdown parser would definitely be more optimal than having this plugin, but that piece of code is needed so they work properly together.